packages/cfd/src/Containers/cfd-dashboard.tsx

```diff
@@ -21,7 +21,7 @@ import MT5TradeModal from './mt5-trade-modal';
 import CFDDbviOnboarding from './cfd-dbvi-onboarding';
 import CFDDownloadContainer from '../Components/cfd-download-container';
 import CFDPasswordManagerModal from './cfd-password-manager-modal';
-import CFDPasswordModal from './cfd-password-modal';
+import CFDPasswordModal from './cfd-password-modal/cfd-password-modal';
```

Comment:
Dont need this change as this file is deleted in another PR

Reply:
I can still see it in master?
https://github.com/binary-com/deriv-app/blob/master/packages/cfd/src/Containers/cfd-password-modal.tsx

🤔

Reply:
Yes you will see it as the PR is not yet merged 😞